zerver/tests/test_events.py

```diff
@@ -3772,6 +3772,8 @@ def test_change_realm_property(self) -> None:
                 self.do_set_realm_permission_group_setting_test(prop)
 
         for prop in Realm.REALM_PERMISSION_GROUP_SETTINGS_WITH_NEW_API_FORMAT:
+            if prop == "can_create_web_public_channel_group":
+                continue
```

Comment:
I guess we should have a comment noting that this setting has limited permissions.

Reply:
Oh, I think this might be better written checking `Realm.REALM_PERMISSION_GROUP_SETTINGS[prop].requires_system_group`.